src/coreclr/gc/env/gcenv.h

```diff
@@ -42,11 +42,10 @@
 
 #define STRESS_LOG
 #ifdef STRESS_LOG
-/*  STRESS_LOG_VA was added to allow sendign GC trace output to the stress log. msg must be enclosed
-      in ()'s and contain a format string followed by 0 - 4 arguments.  The arguments must be numbers or
-      string literals.  LogMsgOL is overloaded so that all of the possible sets of parameters are covered.
-      This was done becasue GC Trace uses dprintf which dosen't contain info on how many arguments are
-      getting passed in and using va_args would require parsing the format string during the GC
+/*  STRESS_LOG_VA was added to allow sending GC trace output to the stress log. msg must be enclosed
+    in ()'s and contain a format string. The arguments must be numbers or string literals.
```

Comment:
I see that you added support for more args but why not mention the number of args it can take like the old comment did? that seems like a useful thing to mention. I'm not sure why you chose to get rid of the mentioning of it?